src/compiler/factory/utilities.ts

```diff
@@ -160,6 +160,8 @@ import {
     ReadonlyKeyword,
     RelationalOperator,
     RelationalOperatorOrHigher,
+    // Will become used when explicit return types are added in declaration file during build step.
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
     SetAccessorDeclaration,

```

Comment:
One thing I'll want to check is that this plays nicely with organizing improts / `eslint-plugin-simple-import-sort`. A good way to test is to swap a few of these import lines around and then see if this comment stays in the right place when fixing it with eslint (or using organize imports with the right settings). I'll check that now.